example/Cpp/src/AlarmClock/AlarmClock.cmake

```
@@ -0,0 +1,8 @@
+find_package (Threads)
+
+set(CROW_HEADERS ../../Crow/include/)
+set(CPP_SOURCES ../../src/)
```

This doesn't add a valid directory. You can simply add the directory in which this files is located:
```suggestion
set(CPP_SOURCES ${CMAKE_CURRENT_LIST_DIR})
```

---

done